Resources/Prototypes/GameRules/events.yml

```diff
@@ -444,10 +444,13 @@
   - type: TraitorRule
   - type: AntagSelection
     definitions:
-    - prefRoles: [ Traitor ]
+    - prefRoles: [ TraitorSleeper ]
       min: 1
```

Comment:
add a fallback role for traitors because that's probably the next best thing if there aren't any sleeper agents.

Reply:
done, tell me if i did it right